### PR TITLE
Add CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by AnandSY**
issue #18 
Added CONTRIBUTING.md file 
It contains steps to contribute to this project.
Please accept this PR under hacktoberfest.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+**11.** Push the committed changes in your feature branch to your remote repo.
+
+```
+git push -u origin <your_branch_name>
+```
+
+**12.** To create a pull request, click on `compare and pull requests`.
+
+**13.** Add appropriate title and description to your pull request explaining your changes and efforts done.
+
```
Comment (SscSPs) [inline CONTRIBUTING.md:68]: Can you add a step here mentioning "In the PR, Add a screenshot of your change visible on UI(if possible)"
This should help reduce the time taken to verify the changes being proposed to be valid and not just spam

**Comment by SscSPs** [comment]
Nice, thanks for this!
Also, can you address the comment? 
Also, it'll be great if you can add a link to this in the README.mdas well

File: CONTRIBUTING.md (hunk 1 above)
Comment (AnandSY) [reply CONTRIBUTING.md:68]: Sure

**Comment by AnandSY** [comment]
I have done both things as you said.
Updated CONTRIBUTING.md file with the step.
Added link to this file in readme under contributions heading.

**Comment by SscSPs** [comment]
Awesome! 
Thanks for the PR, looks good, will merge after the check complete!

**Pull request merged**